internal/repository/repository.go

```diff
@@ -15,6 +15,7 @@ import (
 
 var (
 	ErrApplicationNotFound = fmt.Errorf("application not found")
+	ErrEpochNotFound = fmt.Errorf("epoch not found")
```

Comment:
I guess we could have just one ErrNotFound

Reply:
updated